src/containers/Accounts/AccountAssetTab/AccountAssetTab.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ChangeEvent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { useHistory, useParams } from 'react-router';
+import { AccountTokenTable } from '../AccountTokenTable';
+import { AccountNFTTable } from '../AccountNFTTable/AccountNFTTable';
+
+// TODO: Add state types or convert to react query
+interface Props {
+  account: any;
+}
+
+const AccountAssetTabDisconnected = ({ account }: Props) => {
+  const assetTypes = ['token', 'nft'];
```

Comment:
Would it make sense to rename the asset type here to `issuedToken` so it doesn't get confused with a Non-fungible token when accessed later on?

Reply:
This gets used for the url too.  I can update it be issued.